sdk/core/azure-core/test/ut/proxy_tests/localproxy.passwd/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#@# vim: set filetype=dockerfile:
+# Code taken from: https://github.com/hinata/nginx-forward-proxy
```

Comment:
Even though we are using this just for tests, this _might_ require license attribution in the notice.txt file.

Also, we may want to link to a permalink (specific commit sha) from where we copied this from, in case the source changes/evolves.

Reply:
It turns out that this attribution was stale - I removed it.